### DISCUSSION TITLE
Handle some links in the external default browser

### DISCUSSION THREAD

**Discussion by dzervas** [qutebrowser/qutebrowser#8283] [Q&A] [answered]
I'm trying to create custom wrappers for some web-apps (e.g. discord) and I want any non-`discord.com` domains to open in the default system browser (or a hardcoded command). How can I do that?

**Comment by The-Compiler** [comment, marked as answer]
That's not possible currently (outside of trying to patch around internals in a `config.py`, in which case you're on your own unfortunately). 
 Will probably be once there is a basic plugin API (#30). Relevant point there:

> Use `acceptNavigationRequest`, e.g. to automatically open mpv if a YouTube link is opened.
